Library/Formula/srmio.rb

```diff
@@ -15,7 +15,7 @@ class Srmio < Formula
 
   def install
     if build.head?
-      system "chmod u+x genautomake.sh"
+      chmod 1, "genautomake.sh"
```

Comment:
Yeah. My (limited) experience and the [FileUtils `chmod` doc](http://ruby-doc.org/stdlib-1.9.3/libdoc/fileutils/rdoc/FileUtils.html#method-c-chmod) supports this.

Imho, we should prefer this "symbolic" mode for tweaking permission bits unless you really need to set exact permissions for each of user, group, and world. Lets you do masking operations, and seems more readable. (Same for shell code, btw.)